### DISCUSSION TITLE
This project is on dotnet.reddit.com

### DISCUSSION THREAD

**Discussion by dodyg** [Aguafrommars/TheIdServer#895] [General]
https://www.reddit.com/r/dotnet/comments/ytuhut/theidserver_is_an_actively_maintained_fork_of/

It will be great if you can join the discussion. People have questions.

**Comment by aguacongas** [comment]
This is a completely misinformation, it's not a fork of IdentiyServer4. I reported this thread.  And I don't maintain IS4
